manual/de/ch_Finance-Quote.xml

```diff
@@ -248,6 +254,196 @@ SYNOPSIS
+"ftportfolios_direct" "fundlibrary" "goldmoney" "greece" "hex" "hu" "hufund" "hungary" "hustock"
+"indiamutual" "known_currencies" "lerevenu" "maninv" "morningstar" "morningstarjp" "mstaruk" "nasdaq"
+"nyse" "nz" "nzx" "platinum" "romania" "seb_funds" "sixfunds" "sixshares" "stockhousecanada_fund"
+"tdefunds" "tdwaterhouse" "tiaacref" "tnetuk" "troweprice" "troweprice_direct" "trustnet" "tsp" "tsx"
+"uk_unit_trusts" "ukfunds" "unionfunds" "usa" "usfedbonds" "vanguard" "vwd" "yahoo" "yahoo_asia"
+"yahoo_australia" "yahoo_brasil" "yahoo_europe" "yahoo_json" "yahoo_nz" "yahoo_yql" "za" "za_unittrusts")
+</screen>
+        </para>
+      </formalpara>
+
+      <para>Der erste Teil ist die Fehlermeldung über eine fehlende Mozilla/CA.pm. Am Ende in
+        Klammern steht die normale Ausgabe der aktuell installierten &app-fq;-Version<footnote>
+        <para>Die neueste &app-fq; Version ist 1.52.</para>
+        </footnote>
+        und eine Liste der verfügbaren Kursdaten-Quellen.
+      </para>
+
+      <tip>
+        <para>Wenn ein Fehler angezeigt wird, lesen Sie den nächsten Abschnitt <xref linkend="gnc-fq-update" />.
+          Ansonsten fahren Sie mit <xref linkend="gnc-fq-dump" /> fort.
+        </para>
+      </tip>
+    </sect2>
+
+    <sect2 id="gnc-fq-update">
+      <title>gnc-fq-update</title>
+
+      <abstract>
+        <para>Dieses Programm installiert oder aktualisiert das &app-fq; Softwaremodul und löst bei Bedarf
+          die Abhängigkeiten zu weiteren Softwarepaketen auf.
+          <note>
+            <para>Windows-Benutzer sind im Allgemeinen besser beraten, wenn sie das Update-Tool
+              &gmi.start.win-install-fq; im &gm.start.win; Menü verwenden.
+            </para>
+          </note>
+
+        </para>
+      </abstract>
+
+      <note>
+        <para>Dieses Programm erfordert für eine Installation unter &lin; oder &mac; Superuser-
+          oder Administratorrechte, aber nicht in &win;.
+        </para>
+      </note>
+    </sect2>
+
+    <sect2 id="gnc-fq-dump">
+      <title>gnc-fq-dump</title>
+<!-- Todo: insert link to wiki and/or guide -->
+      <abstract>
+        <para>Dieses Programm liefert Kursdaten für eine Quelle und eine Liste von Symbolen in einem Format,
+          das für Menschen leicht zu lesen ist. Es ist nützlich, um zu überprüfen, ob eine bestimmte
+          Online-Kursquelle erreichbar ist und funktioniert.
+        </para>
+      </abstract>
+      
+      <para>Mit Hilfe diesen Befehls können Sie überprüfen, ob das <xref linkend="tool-ge-Symbol" />,
+        welches Sie für Ihr Wertpapier zum Online-Kursabruf verwenden möchten, bei der
+        gewünschten <xref linkend="tool-ge-TypeQuoteSource" /> funktioniert.
+      </para>
+      
+      <tip>
+        <para>Mit <userinput>gnc-fq-dump</userinput> können Sie Symbole schneller überprüfen, wenn während
+          des Abrufs mit &app; ein Fehler auftritt, weil sich die Ausgabe des Skripts besser zur Analyse eignet.
```

Comment:
… und nicht jedesmal alle funtionierenden Symbole mit abgerufen werden, falls man Kurse abrufen[?] in der Kurs-Datenbank[Verweis?] verwendet. Man denke dabei auch an die Einschränungen der kostenlosen Alphavantage-APIKEY-Variante. [ggf. Verweis auf Anhang o.ä.]